src/test/datascience/interactiveWindow.functional.test.tsx

```diff
@@ -976,7 +976,9 @@ for i in range(100):
             const { window } = await getOrCreateInteractiveWindow(ioc);
             const executed = createDeferred();
             // We have to wait until the execute goes through before we reset.
-            window.onExecutedCode(() => executed.resolve());
+            if (window.onExecutedCode){

```

Comment:
This event is now optional and only implemented by Interactive or webview & not Native Notebooks